rgd/geodata/tests/test_tiles.py

```diff
@@ -18,14 +18,21 @@ def image_entry():
 def test_metadata(api_client, image_entry):
     response = api_client.get(f'/api/geodata/imagery/image_entry/{image_entry.pk}/tiles')
     metadata = response.data
-    assert metadata['levels'] == 2
+    assert metadata['levels'] == 15
```

Comment:
@mcovalt, when I changed the format to PNG this changed to 15. Does that seem okay?